libvcx_core/src/api_vcx/utils/devsetup.rs

```diff
@@ -45,7 +44,7 @@ impl SetupGlobalsWalletPoolAgency {
         let init = Self::init().await;
 
         let pool_name = Uuid::new_v4().to_string();
-        let genesis_path = get_temp_dir_path(DEFAULT_GENESIS_PATH).to_str().unwrap().to_string();
+        let genesis_path = get_temp_file_path(DEFAULT_GENESIS_PATH).to_str().unwrap().to_string();
```

Comment:
Perhaps doing `.into_os_string().into_string().unwrap()` might give very slightly more informative errors (the original `OsString`) than unwrapping an option, but I don't think it matters.

Reply:
I acknowledge the suggestion, I think there might be a lot more places to use `OsString` with more benefit. Since this is just testing, I'll skip it in this pr, but could be nice to do in bulk separately